shared/macros/10-ansible.jinja

```diff
@@ -1544,7 +1549,9 @@ Part of the grub2_bootloader_argument_absent template.
 {{% macro ansible_ensure_authselect_custom_profile() -%}}
 - name: '{{{ rule_title }}} - Get authselect current profile'
   ansible.builtin.shell:
-    cmd: authselect current -r | awk '{ print $1 }'
+    cmd:
+        set -o pipefail
+        authselect current -r | awk '{ print $1 }'
```

Comment:
I think you're creating an invalid YAML, there needs to be `|`  or `|-` after `cmd: ` to create a multiline string.